src/meta_schedule/postproc/verify_gpu_code.cc

```diff
@@ -105,7 +105,6 @@ class VerifyGPUCodeNode : public PostprocNode {
     Target target = context->target.value();
     this->target_constraints_ = Map<String, PrimExpr>{
         {"max_shared_memory_per_block", Extract(target, "shared_memory_per_block")},
-        {"max_local_memory_per_block", Extract(target, "registers_per_block")},
```

Comment:
The local memory restriction is removed to be consistent with AutoScheduler, as discussed with @masahi. CC: @Hzfengsy